mautic/mautic
# Cache implementation in tracking helper

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | No 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The purpose is to make the file 'event' not using a session cookie. The session should be changed to the cache.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Now session is replaced by cache in tracking helper so following functionality need to be verify :-

  - Page tracking 
  - Campaign action `Send tracking event` , This will be available only if  google analytics or Facebook pixel tracking is enabled
  - Campaign action `Show Focus Item`
